roles/openshift_master/tasks/main.yml

```diff
@@ -171,6 +171,14 @@
   when: openshift_master_ha | bool and openshift_master_cluster_method == "native"
 # end workaround for missing systemd unit files
 
+# If the session secrets file was missing from any master, we should
+# clean up the file on masters so that we can get them in sync.
+- name: Delete session secrets file
+  file:
+    path: "{{ openshift.master.session_secrets_file }}"
+    state: absent
+  when: session_secrets_missing | bool
+
```

Comment:
We are already operating on oo_first_master, so we should only check oo_first_master to see if they need to be generated.  We should be able to use something like this for setting the vars for masters other than oo_first_master:
```
openshift_master_session_auth_secrets: "{{ hostvars.groups.oo_first_master.0.session_auth_secret if inventory_hostname not in hostvars.groups.oo_first_master else none }}"
```